package.json

```
@@ -141,8 +141,8 @@
     "diff-match-patch": "1.0.5",
     "dompurify": "3.2.3",
     "double-ended-queue": "2.1.0-0",
-    "engine.io": "^6.5.4",
-    "engine.io-client": "^6.5.3",
+    "engine.io": "6.5.5",
```

Please, keep the `^` (aka the exact version)

```suggestion
    "engine.io": "^6.5.5",
```

---

I will pin both to explicit version.
Thanks for the suggestion :)